enterprise/lms_api.py

```diff
@@ -24,24 +26,36 @@
     CourseEnrollment = None
 
 
+LOGGER = logging.getLogger(__name__)
 LMS_API_DATETIME_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
 
 
-class EnrollmentApiClient(object):
+class LmsApiClient(object):
```

Comment:
@e-kolpakov `LmsApiClient` actually functions on its own, since the Slumber API is so flexible, so I'm inclined to leave it as concrete.